components/RolesSection.tsx

```diff
@@ -0,0 +1,158 @@
+import CalculateOutlinedIcon from '@mui/icons-material/CalculateOutlined'
+import ChecklistOutlinedIcon from '@mui/icons-material/ChecklistOutlined';
+import CodeOutlinedIcon from '@mui/icons-material/CodeOutlined'
+import DescriptionOutlinedIcon from '@mui/icons-material/DescriptionOutlined'
+import Diversity3OutlinedIcon from '@mui/icons-material/Diversity3Outlined'
+import DrawOutlinedIcon from '@mui/icons-material/DrawOutlined'
+import EmojiPeopleOutlinedIcon from '@mui/icons-material/EmojiPeopleOutlined'
+import GavelRoundedIcon from '@mui/icons-material/GavelRounded'
+import HubOutlinedIcon from '@mui/icons-material/HubOutlined';
+import InsightsOutlinedIcon from '@mui/icons-material/InsightsOutlined'
+import IntegrationInstructionsOutlinedIcon from '@mui/icons-material/IntegrationInstructionsOutlined';
+import ManageAccountsOutlinedIcon from '@mui/icons-material/ManageAccountsOutlined'
+import PeopleOutlineOutlinedIcon from '@mui/icons-material/PeopleOutlineOutlined'
+import ScreenSearchDesktopOutlinedIcon from '@mui/icons-material/ScreenSearchDesktopOutlined'
+import ShareOutlinedIcon from '@mui/icons-material/ShareOutlined'
+import { Box, Button } from '@mui/material'
+import { ReactNode } from 'react'
+import { DASVolunteerRoleBasicInfo } from 'types'
+
+import ListItemWithIcon from './list/ListItemWithIcon'
+import { Section, Subheader } from './style-utils'
+
+// TODO: standardize roles between sanity and airtable
+const rolesMap = {
+  communityEngagementLiason: { role: 'community engagement liaison', icon: <CampaignOutlinedIcon />, },
+  dataAnalyst: { role: 'data analyst', icon: <ScreenSearchDesktopOutlinedIcon />, },
+  designer: { role: 'designer', icon: <DrawOutlinedIcon />, },
+  graphicDesigner: { role: 'graphic designer', icon: <DrawOutlinedIcon />, },
+  grantWriter: { role: 'grant writer', icon: <DescriptionOutlinedIcon />, },
+  legalHelp: { role: 'legal help', icon: <GavelRoundedIcon />, },
+  productManager: { role: 'product manager', icon: <Diversity3OutlinedIcon />, },
+  projectManager: { role: 'project manager', icon: <ManageAccountsOutlinedIcon />, },
+  uxResearcher: { role: 'user experience researcher', icon: <ScreenSearchDesktopOutlinedIcon />, },
+  userExperienceResearcher: { role: 'user experience researcher', icon: <ScreenSearchDesktopOutlinedIcon />, },
+  uxUiDesigner: { role: 'UX/UI designer', icon: <ScreenSearchDesktopOutlinedIcon />, },
+  socialMediaDesigner: { role: 'social media designer', icon: <ShareOutlinedIcon />, },
+  socialMediaSpecialist: { role: 'social media specialist', icon: <EmojiPeopleOutlinedIcon />, },
+  softwareEngineer: { role: 'software engineer', icon: <CodeOutlinedIcon />, },
+  softwareDeveloper: { role: 'software developer', icon: <CodeOutlinedIcon />, },
+  solutionArchitect: { role: 'solution architect', icon: <ApartmentOutlinedIcon />, },
+  qaTestEngineer: { role: 'QA/test engineer', icon: <BugReportOutlinedIcon /> },
```

Comment:
Do we really need two roles for a QA? See QA test engineer (#61) and QA specialist (#63)?

Reply:
@ybalenko, some of these are roles that are leftover from using Sanity. If we eventually get rid of the icons on the volunteer role cards, this whole map can go away.